Makefile

```
@@ -216,8 +216,17 @@ license-check: ## Check license is approved
         --ignore=github.com/hashicorp/hcl \
         --ignore=$$STDLIB_IGNORE
 
+.PHONY: test-shell
+test-shell: ## Runs shell unit tests (tools/*_test.sh; hermetic, no cluster)
```

checkmake (0.3.2)

[warning] Target body for "test-shell" exceeds allowed length of 5 lines (7).

(maxbodylength)